_articles/getting-paid.md

```diff
@@ -104,11 +104,16 @@ If you can't convince your current employer to prioritize open source work, cons
 
 Projects that originated at a large company, such as [Go](https://github.com/golang) or [React](https://github.com/facebook/react), will also likely employ people to work on open source.
 
-Finally, depending on your personal circumstances, you can try raising money independently to fund your open source work. For example:
+Depending on your personal circumstances, you can try raising money independently to fund your open source work. For example:
 
 * @gaearon funded his work on [Redux](https://github.com/reactjs/redux) through a [Patreon crowdfunding campaign](https://redux.js.org/)
 * @andrewgodwin funded work on Django schema migrations [through a Kickstarter campaign](https://www.kickstarter.com/projects/andrewgodwin/schema-migrations-for-django)
 
+Finally, there are also open source projects which put bounties on issues that you might consider helping with.  Many of the organizations funding their projects also use bounty platforms for recruiting. For Example: 
```

Comment:
Is there a source for the use for recruiting claim?

"Example" should be "example".

Reply:
There should only be one space between sentences (remove one before "Many").

Reply:
Ah, thank you for catching that! I wasn't quite sure what was causing the build to fail, I will fix it.

An _example_ is that on the results page of gitcoin it shows that "5 GitCoin Bounty Hunters Hired Full Time"
https://gitcoin.co/results

Reply:
That seems like pretty weak evidence for the claim made in the "Many..." sentence. I'd suggest removing that sentence unless there's more evidence it is true.

Reply:
Okay, fair enough. I guess I thought it was more prevalent. I looked up bountysource too.. it seems they don't have a jobs board either so I will remove that line